platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -192,26 +184,30 @@ class Viewer extends Component {
       const patientId = studies[0] && studies[0].patientId;
 
       timepointApi.retrieveTimepoints({ patientId });
-      measurementApi.retrieveMeasurements(patientId, [currentTimepointId]);
-
+      if (this.props.studyLoaded) {
+        this.measurementApi.retrieveMeasurements(patientId, [
+          currentTimepointId,
+        ]);
+      }
```

Comment:
I wonder if this makes more sense to do with promises/events instead of in mounted and updated w/ a reactive bool.